api/cron-jobs/projects.js

```diff
@@ -129,51 +153,88 @@ function formatProjects(projects, resources) {
 }
 
 async function getAllProjectsAndResourcesFromJira() {
-  console.log('🛈 Getting data from Jira and Vimeo APIs - this can take 5 seconds or more');
+  console.log(
+    '🛈 Getting data from Jira and Vimeo APIs - this can take 5 seconds or more'
+  );
 
   return new Promise((resolve) => {
-    const callAllItData = Promise.resolve(module.exports.getInitialTriageProjectsFromJira(0, []));
-    const callAllResData = Promise.resolve(module.exports.getResourcesFromJira(0, []));
+    const callAllItData = Promise.resolve(
+      module.exports.getInitialTriageProjectsFromJira(0, [])
+    );
+    const callAllResData = Promise.resolve(
+      module.exports.getResourcesFromJira(0, [])
+    );
 
     Promise.all([callAllItData, callAllResData]).then((data) => {
-      const itArray = data[0];
-      const resArray = data[1];
-
-      const projectsFiltered = module.exports.filterProjectsConnectedWithResources(itArray, resArray);
-      const resourcesFiltered = module.exports.filterResourcesConnectedWithProjects(itArray, resArray);
-      const projectsFilteredAndFormatted = module.exports.formatProjects(projectsFiltered, resourcesFiltered);
-
-      console.log(
-        `🛈 Found ${projectsFilteredAndFormatted.length} projects matching resources, ${resourcesFiltered.length} resources matching projects`,
-      );
-
-      resolve({
-        projects: projectsFilteredAndFormatted,
-        resources: resourcesFiltered,
-      });
+      try {
+        const itArray = data[0];
+        const resArray = data[1];
+
+        if (data[0] === undefined || data[1] === undefined)
```

Comment:
Small thing and it doesn't change the result, but it might be clearer here if this could be `if (itArray === undefined || resArray === undefined)`

Reply:
Actually, a shorter, neater way of writing this would simply be `if (!itArray || !resArray)` (if one of these is undefined this check will catch it)